ClipboardAPI/clipboard-change-event-example-app.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Clipboard Format Checker</title>
+    <script>
+        const isClipboardChangeEventAvailable = true;
+
+        async function checkClipboard() {
+            try {
+                console.log("Clipboard changed!");
+                const clipboardItems = await navigator.clipboard.read();
+                let formats = {};
+
+                for (const item of clipboardItems) {
+                    if (item.types.includes('text/plain')) {
+                        const textBlob = await item.getType('text/plain');
+                        const text = await textBlob.text();
+                        document.getElementById('clipboardText').innerText = text;
+                        formats.text = true;
+                    }
+                    else{
```

Comment:
nit: Let's be consistent with our style for brackets and spaces everywhere. Ie. `if () {` and `else {`